File: PerformanceTesting/GLCubeResult.cs
using System;
namespace PerformanceTesting
{
	public class GLCubeResult
	{
		public GLCubeResult ()
		{
		}
		
		public GLCubeResult (int numberOfTriangles, double framePerSecond)
		{
			this.NumberOfTriangles = numberOfTriangles;
			this.FramesPerSecond = framePerSecond;
		}
		
		public int NumberOfTriangles {get;set;}
		public double FramesPerSecond {get;set;}
	}
}

File: CPerformanceTesting/CPerformanceTesting/GLCubeResult.m
//
//  GLCubeResult.m
//  CPerformanceTesting
//
//  Created by William Holmes on 6/24/12.
//  Copyright (c) 2012 Slap Holmes Productions. All rights reserved.
//

#import "GLCubeResult.h"

@implementation GLCubeResult

@synthesize numberOfTriangles;
@synthesize framesPerSecond;

- (id)initWithNumberOfTriangles: (int) numTriangles framesPerSecond: (double) fps
{
    self = [super init];
    if (self) {
        self.numberOfTriangles = numTriangles;
        self.framesPerSecond = fps;
    }
    return self;
}

@end
